pages/components/LoginComponent.js

```diff
@@ -0,0 +1,101 @@
+import logo from "../../assets/chat.png";
+import Image from 'next/image';
+import Link from 'next/link'
+import { Container,LogoWrapper,Form, StyledInput, InputContainer, Errors} from "../../styledComps/artifacts";
+import { useRouter } from "next/router";
+import { useState,useEffect } from "react";
+import { useFormik } from 'formik';
+import axios from "axios";
+import { SERVER_URL } from "../../server";
+ import * as Yup from 'yup';
+ 
+const LoginComponent = () => {
+  const router = useRouter()
+  const [token,setToken]=useState('');
+  const [apiErrors,setApiErrors]= useState("")
+
+  useEffect(() => {
+    //console.log("Token from useEffect from login",token)
```

Comment:
make sure to remove commented code and console statements before pushing the code.